config/src/networks.rs

```diff
@@ -251,6 +268,45 @@ pub fn holesky() -> BaseConfig {
     }
 }
 
+pub fn devnet(config: ChainConfig) -> BaseConfig {
+    // This configuration needs to be updated every time you spin up a local eth network (genesis_time, genesis_root)
+    BaseConfig {
+        default_checkpoint: hex_str_to_bytes(
+            "0xb3b5d384704782ab9e17969ad692340b1f8952baf5e8089b8317fc45f6b360e3",
+        )
+            .unwrap(),
+        rpc_port: 8545,
+        consensus_rpc: Some("http://localhost:3500".to_string()),
+        chain: config.clone(),
+        forks: Forks {
+            genesis: Fork {
+                epoch: 0,
+                fork_version: hex_str_to_bytes("0x20000089").unwrap(),
+            },
+            altair: Fork {
+                epoch: 0,
+                fork_version: hex_str_to_bytes("0x20000090").unwrap(),
+            },
+            bellatrix: Fork {
+                epoch: 0,
+                fork_version: hex_str_to_bytes("0x20000091").unwrap(),
+            },
+            capella: Fork {
+                epoch: 0,
+                fork_version: hex_str_to_bytes("0x20000092").unwrap(),
+            },
+            deneb: Fork {
+                epoch: 132608,
+                fork_version: hex_str_to_bytes("0x20000093").unwrap(),
+            },
+        },
+        max_checkpoint_age: 1_209_600, // 14 days
+        #[cfg(not(target_arch = "wasm32"))]
+        data_dir: Some(data_dir(Network::DEVNET(config))),
+        ..std::default::Default::default()
+    }
+}
```

Comment:
This function cannot be removed since we use it for local testing & e2e testing of the light client with dwallet network and a local ethereum network.
I think the best way to do this is by taking a path to an ethereum network config file and load all data from there.

Reply:
As you would, but it cannot remain as it is now